tests/infra/bencher.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import os
+import json
+
+BENCHER_FILE = "bencher.json"
+
+class Bencher:
+    def __init__(self):
+        if not os.path.isfile(BENCHER_FILE):
+            with open(BENCHER_FILE, "w+") as bf:
+                json.dump({}, bf)
+    
+    def set(self, key, value):
```

Comment:
This looks great! Can we add a type for `value` to help build something in the format bencher expects (or at least a link to the intended schema near here)?